dist/esm/index.d.ts

```diff
@@ -1,2 +1,2 @@
-import './helpers.ts';
```

Comment:
Note that I've reverted this change since we shouldn't do manual changes to generated files. Did your editor tell you ESLint issues or you did run some command?

Reply:
Oops, this must have happened when I ran `npm run build` and committed those changes. I definitely didn't manually do this.